odysseus/odysseus_tree/overlays/rootfs_overlay_tpu/etc/hostapd.conf

```diff
@@ -0,0 +1,20 @@
+ctrl_interface=/var/run/hostapd
+country_code=US
```

Comment:
Kinda crazy how similar this config is to HaLow config lol

Reply:
ya nrc does a pretty good job of making nrc like a normal network card.